packages/pwa-kit-react-sdk/src/storefront-preview/index.jsx

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/**
+ * Create alias import path for StorefrontPreview component
+ * to indicate this is a first-class feature
+ */
+import {StorefrontPreview} from '../ssr/universal/components/storefront-preview'
```

Comment:
Hmm.. this file should be deleted, because we already moved the StorefrontPreview component to a different package now, commerce-sdk-react.